### ISSUE TITLE
wrong with images

### ISSUE THREAD

**Issue by demo1shining** [Junjue-Wang/EarthVQA#3]
![2617](https://github.com/Junjue-Wang/EarthVQA/assets/99237507/cb5e971a-a9c9-4034-9798-79b260d90af0)
![2621](https://github.com/Junjue-Wang/EarthVQA/assets/99237507/77f44c57-0164-4f4c-b625-2cd087dc3fa6)
![2615](https://github.com/Junjue-Wang/EarthVQA/assets/99237507/d4e96e75-b1c1-462a-a759-01588c714b01)
When I download images from Google, some images are partially missing. This problem exists in the training set, validation set and test set.

**Comment by Junjue-Wang** [comment]
These images are located on the edge of one large-scale region. The no-data has no effect during the training when the supervised label is set to -1.

**Comment by demo1shining** [comment]
Okay, thanks for the answer！
